getting-started/initialize-the-sdk.md

```diff
@@ -103,6 +103,56 @@ ACPCore.setLogLevel(ACPMobileLogLevel.debug)
 {% endtab %}
 {% endtabs %}
 
+## Registering Extensions and Starting Core
+
+All AEP extensions provide a `registerExtension` API (except Core), this API registers the extension with Core which allows the extension to dispatch and listen for events. You are required to register each of your extensions before making API calls and failing to do so will lead to undefined behavior.
```

Comment:
Please change the 1st sentence so it reads as follows:

"Other than the Mobile Core extension, all Experience Platform extensions provide a `registerExtension` API, which registers the extension with Core. After you register the extension, you can dispatch and listen for events."

Reply:
This sounds much better, thanks.